src/azure-cli/azure/cli/command_modules/monitor/operations/general_operations.py

```diff
@@ -0,0 +1,61 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from .._client_factory import cf_monitor
+from ..util import _gen_guid
+from azure.cli.core.commands.transform import _parse_id
+from knack.log import get_logger
+from knack.util import CLIError
+from msrestazure.tools import parse_resource_id
+
+
+logger = get_logger(__name__)
+
+
+def clone_existed_settings(cmd, source_resource=None, target_resource=None):
+    metrics_alert_rules = _clone_monitor_metrics_alerts(cmd, source_resource, target_resource)
+    return metrics_alert_rules
+
+
+def _clone_monitor_metrics_alerts(cmd, source_resource, target_resource):
+    if not _is_resource_type_same(source_resource, target_resource):
+        raise CLIError('The target resource should be the same type with the source resource')
+    monitor_client = cf_monitor(cmd.cli_ctx)
+    # we can only clone the alert rules belonging to the current subscription.
+    alert_rules = list(monitor_client.metric_alerts.list_by_subscription())
```

Comment:
According to here, you only list alert rules in one subscription, right? And the concern is the same as above.

Reply:
yes.